contract/views/account_analytic_account_view.xml

```diff
@@ -1,22 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <record id="act_recurring_invoices" model="ir.actions.act_window">
-        <field name="context">{'search_default_contract_id':
-        [active_id],
-        'default_contract_id': active_id}
-        </field>
-        <field name="name">Invoices</field>
-        <field name="res_model">account.invoice</field>
-        <field name="view_id" ref="account.invoice_tree" />
-        <field name="search_view_id" ref="account.view_account_invoice_filter"/>
-    </record>
-
     <record id="account_analytic_account_recurring_form_form" model="ir.ui.view">
         <field name="name">account.analytic.account.invoice.recurring.form.inherit</field>
         <field name="model">account.analytic.account</field>
         <field name="inherit_id" ref="analytic.view_account_analytic_account_form"/>
-        <field eval="40" name="priority"/>
+        <field name="mode">primary</field>
```

Comment:
Note setting the mode to primary seemed more appropriate here. In the previous case this view would be appended to all `account.analytic.account` views, which I don't believe was the intent. Please correct me if I'm wrong on that assumption.